frontend/jest.config.cjs

```
@@ -31,99 +13,21 @@ const config = {
-    '^.+\\.tsx?$': [
-      'ts-jest',
-      {
-        useESM: true,
-        tsconfig: {
-          jsx: 'react-jsx',
-          esModuleInterop: true,
-          allowSyntheticDefaultImports: true,
-        },
-      },
-    ],
+    'os-platform/core/canon/layoutEnvelope\.mjs$': path.resolve(__dirname, '..', 'os-platform', 'core', 'canon', 'layoutEnvelope.jest.cjs'),
```

ESLint

[error] Unnecessary escape character: \..

(no-useless-escape)